cql2pgjson/src/main/java/org/folio/cql2pgjson/util/DbSchemaUtils.java

```diff
@@ -43,6 +47,13 @@ private static boolean checkDbIndex(String cqlIndex, List<Index> indexes) {
     return getIndex(cqlIndex, indexes) != null;
   }
 
+  /**
+   * Get {@link Index} for given cqlIndex name.
+   *
+   * @param cqlIndex
+   * @param indexes
+   * @return
```

Comment:
Can you document what is returned if not found?

Reply:
done